### ISSUE TITLE
Build error: global/src/scalapack.F This name does not have a type, and must have an explicit type.

### ISSUE THREAD

**Issue by FreemanTheMaverick** [GlobalArrays/ga#238]
Hi. I am building GA with Intel tool set. The configuration is:
```
./configure MPICXX=mpiicpc MPIF77=mpiifort MPICC=mpiicc --prefix=/scratch/PI/yonghuang/GlobalArray --enable-i8 --with-sockets --with-blas8="-L$MKLROOT/lib/intel64 -lmkl_intel_ilp64 -lmkl_intel_thread -lmkl_core -liomp5 -lpthread -lm" --with-scalapack8="-L$MKLROOT/lib/intel64 -lmkl_scalapack_ilp64 -lmkl_intel_ilp64 -lmkl_intel_thread -lmkl_core -lmkl_blacs_intelmpi_ilp64 -liomp5 -lpthread -lm"
```
And the MPI is:
```
$ mpirun -V
Intel(R) MPI Library for Linux* OS, Version 2019 Update 7 Build 20200312 (id: 5dc2dd3e9)
Copyright 2003-2020, Intel Corporation.
```
When I `make -j`, errors show up like this:
```
global/src/scalapack.F(2454): error #6404: This name does not have a type, and must have an explicit type.   [GA_DEALLOCATE]
      if (.not.ga_deallocate(g_b)) 
---------------^
global/src/scalapack.F(3125): error #6404: This name does not have a type, and must have an explicit type.   [GA_DEALLOCATE]
           if(.not.ga_deallocate(g_b))
-------------------^
global/src/scalapack.F(3405): error #6404: This name does not have a type, and must have an explicit type.   [GA_DEALLOCATE]
         if(.not.ga_deallocate(g_b))
-----------------^
compilation aborted for global/src/scalapack.F (code 1)
```
I would appreciate it if anyone could help me with this.

**Comment by FreemanTheMaverick** [comment]
I have solved this by using an older version, v5.7.2.

I am going to leave this issue open until it is truly solved in v5.8.

**Comment by edoapra** [comment]
The tarball has been updated with fix https://github.com/GlobalArrays/ga/commit/a84cb63584204a22efa806e9e1c2ea10688d57bc
